docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -28,6 +28,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'SlimChess - Play Instant Chess Online Games on the Go',
```

Comment:
In my opinion shouldn't it be the other way - newly added should be in the front as it showcases the latest features / changes of the library?